big_tests/tests/pubsub_SUITE.erl

```diff
@@ -352,7 +354,17 @@ subscribe_unsubscribe_test(Config) ->
               pubsub_tools:subscribe(Bob, Node, [{jid_type, bare}]),
               pubsub_tools:unsubscribe(Bob, Node, [{jid_type, bare}]),
 
-              pubsub_tools:delete_node(Alice, Node, [])
+              pubsub_tools:delete_node(Alice, Node, []),
+
+              BobJid = escalus_utils:get_jid(Bob),
+              instrument_helper:assert(mod_pubsub_set_subscribe, #{host_type => domain()},
+                                       fun(#{count := 1, jid := From, time := T}) ->
+                                           T >= 0 andalso BobJid =:= jid:to_binary(From)
+                                       end),
```

Comment:
How about a helper?

```
assert_event(EventName, Client) ->
   ClientJid = escalus_utils:get_jid(Client),
   instrument_helper:assert(EventName, #{host_type => domain()},
                            fun(#{count := 1, jid := From, time := T}) ->
                                T >= 0 andalso BobJid =:= jid:to_binary(From)
                            end),
```

Reply:
Good idea, thanks! I've also added an async version. There is one place which checks for errors using a different fun, but I left it as it is now.